sources/api/migration/migrator/src/main.rs

```diff
@@ -479,10 +490,10 @@ where
 }
 
 fn load_manifest(repository: &tough::Repository) -> Result<Manifest> {
-    let target = "manifest.json";
+    let target = "manifest.json".try_into().unwrap();
```

Comment:
Did you forget to use an error variant here?

Reply:
Because `"manifest.json"` is a constant, we know it can't possibly return an error here, so we just went with `unwrap()`. Should apply to any other addition of unwrap in this PR.

Reply:
I ended up adding `context()` calls to these constant unwraps, which should make clippy lints more meaningful, which I think is worth it.